plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -300,15 +303,21 @@ def _process_manifest(self, manifest, digest, available_blobs, tag=None):
             manifest_file.write(manifest)
         manifest = models.Manifest.from_json(manifest, digest)
         self.parent.available_manifests.append(manifest)
+        foreign_layer = 'application/vnd.docker.image.rootfs.foreign.diff.tar.gzip'
+        has_foreign_layer = False
         for layer in manifest.fs_layers:
-            available_blobs.add(layer.blob_sum)
+            if layer.layer_type and layer.layer_type == foreign_layer:
```

Comment:
So the value would be an empty string, or ``None``? Either way, that would not be equal to ``foreign_layer``, right?